test/AutoBus.spec.ts

```diff
@@ -0,0 +1,104 @@
+import AutoBus from '../src/AutoBus';
```

Comment:
Are these tests even run? The jest config probably doesn't know about `test` as a source root.

Reply:
I think we should move the test (spec) files into `src`, as siblings of the source files.

Reply:
Yes, I changed `roots` config in `jest.config.js`